code/hud/hudreticle.cpp

```diff
@@ -309,12 +309,19 @@ ship_info *sip = &Ship_info[Player_ship->ship_info_index];
 }
 
 void HudGaugeReticle::getFirepointStatus() {
+
+	// allow the firepoint status to be empty when a multiplayer observer
+	// this is not a bug, the observer will simply *not* have any firepoints.
+	if (Objects[Player->objnum].type == OBJ_OBSERVER) {
+		return; 
```

Comment:
Should we also assert here that `Game_mode & GM_MULTIPLAYER`?

Reply:
That is a great idea! :D